docs/_posts/1871-05-19-Louis-Rau.md

```diff
@@ -5,27 +5,27 @@ author: Louis Rau
 layout: post
 place: Brussel
 lang: 'de'
-signature: 'Eure Ludwig'
+signature: 'Euren Ludwig'
 scan_image: '1871-05-19_Brussels_Ludwig_Rau.jpg'
 note: (Au dos de [la lettre des 16 et 17 Mai 1871](/1871/05/17/Louis-Guillier.html))
 ---
 
 Meine Theuren,
 
 Beifolgend 2 Bf von
-<span data-lang="fr">Guillier</span>; es ist schauderschrift mir es
+<span data-lang="fr">Guillier</span>; es ist schauderhaft wie es
 in <span data-lang="fr">Paris</span> ausschaut & man weis
-gar nicht wan das reden wird.
+gar nicht wan̄ das enden wird.
 
-Einstweilen kann ich mich sie
-wie wenig beschaufliegen, und
-denke ich d?dung wenige an unse
-<span data-lang="fr">Schlamassel</span> : — Wir sehen Euen
+Einstweilen kann ich mich hier
+nur wenig beschäftigen, und
+denke ich dadurch weniger an unser
+<span data-lang="fr">Schlamassel</span> : — Wir sehnen Euren
 guten Negrenschen betreffs Landerhaus
```

Comment:
I'd like to read “Nachrichten” but there are too many strokes at the end of the word. However, I'm quite sure about the beginning “Nach”. 

And the last word on the line could be “Landwohnung”.

Reply:
Yeah. I ended up with "Negrenschen" because I couldn't read anything else, and the Grünberg family had African servants from Alexandria.